easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-8-foss-2020b-20210316.eb

```diff
@@ -0,0 +1,45 @@
+name = 'OpenFOAM'
+local_commit = '1c9b587'
+version = '8'
+versionsuffix = '-20210316'
```

Comment:
@hajgato it seems a bit odd to have this as the `versionsuffix`, rather than being part of the version?

Reply:
I guess that's sort of true, but we used the same approach in `OpenFOAM-7-foss-2019b-20200508.eb`.

The `-20210316` part is something we came up with, it's not part of an official OpenFOAM release, so using `8-20210316` may be misleading?

Reply:
I don't have strong feelings about it, so we might as well stick with the previous approach.